internal/cmd/egctl/testdata/translate/out/quickstart.all.yaml

```diff
@@ -87,6 +87,7 @@ infraIR:
         labels:
           gateway.envoyproxy.io/owning-gateway-name: eg
           gateway.envoyproxy.io/owning-gateway-namespace: envoy-gateway-system
+          gateway.networking.k8s.io/gateway-name: eg
```

Comment:
EG differs from the GEP in two ways
* label name `gateway.envoyproxy.io/owning-gateway-name`
* generated resource is in the same ns as the Gateway

this warrants a bigger discussion, lets bring this up in the community meeting next week

Reply:
This PR only adds the required label `gateway.envoyproxy.io/owning-gateway-name` as it's compatible with the current EG implementation.

The existing `gateway.envoyproxy.io/owning-gateway-name` is reserved as it's used as a selector to filter out services and deployments related to a Gateway.